src/all/akuma/src/eu/kanade/tachiyomi/extension/all/akuma/Akuma.kt

```diff
@@ -268,6 +263,9 @@ class Akuma(
                 SChapter.create().apply {
                     url = "${manga.url}/1"
                     name = "Chapter"
+                    date_upload = SimpleDateFormat("yyyy-MM-dd HH:mm", Locale.ENGLISH).apply {
+                        timeZone = TimeZone.getTimeZone("UTC")
+                    }.parse(client.newCall(chapterListRequest(manga)).execute().asJsoup().select(".date .value>time").text())?.time!!
```

Comment:
just remove override for `fetchChapterList` and provide implementation in `chapterListParse` where you will get the response directly